package.json

```diff
@@ -15,6 +15,10 @@
     {
       "name": "Nimisha",
       "email": "nmukherj@redhat.com"
+    },
+    {
+      "name": "Sudipta Sen",
+      "email": "sudsen@redhat.com"
```

Comment:
While this is very important to add, it should go in it's own PR. It is equally important to keep all changes focused on a single task. That way if we every have to role it back we don't need to worry about losing something like this.